src/Sensorario/DiamondKata/DiamondKata.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Sensorario\DiamondKata;
+
+final class DiamondKata
+{
+    const EOL = "\n";
+
+    private $params;
+
+    private function __construct(array $params)
+    {
+        $this->params['input'] = $params['input'];
+    }
+
+    public static function createFrom($input)
+    {
+        return new self([
+            'input' => $input
+        ]);
+    }
```

Comment:
Why a private construct and a static "createForm" method instead of a simple constructor? At first glance it confused me a bit.